charts/base-cluster/templates/nfs-server-provisioner/rules/storage-size.yaml

```
@@ -9,7 +9,7 @@ metadata:
   name: storage-size
   namespace: nfs-server-provisioner
   labels: {{- include "common.labels.standard" $ | nindent 4 }}
-    {{- with .Values.monitoring.labels }}{{- include "common.tplvalues.render" (dict "value" . "context" $) | nindent 4 }}{{- end }}
+    monitoring/provisioned-by: base-cluster
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)